provisioning-telephony/nodejs/index.js

```diff
@@ -0,0 +1,331 @@
+// Import json file that will be used later for creating objects.
+const inputTemplate = require('./input-template.json');
+// Set Genesys cloud objects
+const platformClient = require('purecloud-platform-client-v2');
+const client = platformClient.ApiClient.instance;
+// Instantiate APIs
+const authorizationApi = new platformClient.AuthorizationApi();
+const telephonyProvidersEdgeApi = new platformClient.TelephonyProvidersEdgeApi();
+const locationsApi = new platformClient.LocationsApi();
+
+// Declare global variables
+let locationInfo = null;
+let siteId = null;
+
+// Get client credentials from environment variables
+const CLIENT_ID = inputTemplate.client_id;
```

Comment:
Instead of using values from the template, can you use `process.env.GENESYS_CLIENT_ID` AND `process.env.GENESYS_CLIENT_SECRET` here? The best practice for handling a client secret is to not store it at rest, like in a config file. Setting it as an environment variable helps reduce accidental leakage.

Reply:
this has been updated.